draft-ietf-rats-architecture.md

```diff
@@ -330,14 +332,15 @@ implicitly trusted is often referred to as a Root of Trust.
 It is important to prevent replay attacks where an attacker replays
 old Evidence or an old Attestation Result that is no longer correct.
 To do so, some mechanism of ensuring that the Evidence and Attestation
-Result are fresh, meaning that there is some degree of assurance
+Results are fresh, meaning that there is some degree of assurance
```

Comment:
Intuitively freshness is the time since the request was made and the response was received. Do we need more than this?